salesagility/SuiteCRM
# Fix/5073 aor report conditions

Fixes two bugs in the AOR report parameters:
A duplicate line being created for each parameter value.
Dates already in the correct dabase format being converted again (and failing).

## Description



Issue #5073 

## How To Test This


1. Create a custom date field on Invoice
2. Create a report related to Invoice, with any fields
3. Add condition Assigned to X user
4. Add condition Status: Paid
5. And then one of the following conditions
- Custom date greater than AND Custom date lesser than
- Custom date equal to period
6. Mark all the conditions as Parameter
7. Visualize the report
8. Make sure the report shows correctly even when changing the Parameters and hitting Update

## Types of changes


### Final checklist
